build.gradle

```diff
@@ -49,14 +50,7 @@ task updateInstaller(type: Copy, dependsOn: assemble, group: "Binaries", descrip
     rename "jFSTMerge.*", "s3m.jar"
     
     doLast {
-        File installer = file("installer/s3mInstaller.jar");
-        if(!installer.exists() || installer.length() < 5000) {
-            exec {
-                commandLine("git", "lfs", "pull", "-I", "installer/s3mInstaller.jar")
-            }
-        }
-
-        ant.jar(update: "true", destfile: installer) {
+        ant.jar(update: "true", destfile: "installer/s3mInstaller.jar") {
```

Comment:
The rest of this task is no longer here. If the installer binary isn't available (after a git clone with lfs, for example), the build will never succeed.